src/test/java/core/basesyntax/operations/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,47 @@
+package core.basesyntax.operations;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.storage.Storage;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static BalanceOperationHandler operationHandler;
+    private static Fruit apple;
+    private static Fruit banana;
+
+    @BeforeClass
+    public static void beforeClass() {
+        Storage.storage.clear();

```

Comment:
Why do you use clear? If you use annotation @BeforeClass this code will be executed once, when you run the test. You clear an empty map because you fill it on 25 line